vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -20080,6 +20080,19 @@ quantitykind:SpecificInternalEnergy
   rdfs:seeAlso quantitykind:SpecificGibbsEnergy ;
   rdfs:seeAlso quantitykind:SpecificHelmholtzEnergy ;
 .
+quantitykind:SpecificModulus
+  a qudt:QuantityKind ;
+  dcterms:description """<p><b>Specific modulus</b> is a <a href=\"/wiki/Materials_property\" class=\"mw-redirect\" title=\"Materials property\">materials property</a> consisting of the <a href=\"/wiki/Elastic_modulus\" title=\"Elastic modulus\">elastic modulus</a> per mass <a href=\"/wiki/Density\" title=\"Density\">density</a> of a material. It is also known as the <b>stiffness to weight ratio</b> or <b>specific stiffness</b>. High specific modulus materials find wide application in <a href=\"/wiki/Aerospace\" title=\"Aerospace\">aerospace</a> applications where minimum structural <a href=\"/wiki/Weight\" title=\"Weight\">weight</a> is required. The <a href=\"/wiki/Dimensional_analysis\" title=\"Dimensional analysis\">dimensional analysis</a> yields units of distance squared per time squared. The equation can be written as:
```

Comment:
Actually, the dcterms:description kind of leaves me hanging at the very end. Perhaps delete the last sentence? The reader can still consult the latexDefinition...

Reply:
Apparently the equation didn't copy correctly. But you're right, the latex definition has it. Fixed.